tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -135,6 +135,9 @@ describe(__filename, () => {
     return shallowUntilTarget(
       <AddonsByAuthorsCard {...props} />,
       AddonsByAuthorsCardBase,
+      {
+        shallowOptions: createContextWithFakeRouter({ location }),
```

Comment:
This seems pretty common. Could we add a new option to `shallowUntilTarget` like `{ withFakeRouter: true }` or something?

Reply:
Not sure because we often want to pass something. Would it add some value?

Reply:
oh, I thought it was not important to reference `location` in most cases. I guess that wouldn't be helpful then.